sdk/scheduler/src/main/java/org/apache/mesos/scheduler/plan/DefaultStep.java

```diff
@@ -137,16 +105,18 @@ public Status getStatus() {
     }
 
     @Override
-    public List<Element> getChildren() {
+    public List<Element<?>> getChildren() {
         return Collections.emptyList();
     }
 
     @Override
-    @SuppressWarnings("unchecked")
-    public Strategy<? extends Element> getStrategy() {
+    public Strategy<Step> getStrategy() {
```

Comment:
A Phase has a strategy to deploy Steps.  A Step doesn't have Steps as children.  I suppose this gets rid of the unchecked warning...